lib/lightning_web/live/components/menu.ex

```diff
@@ -57,74 +57,82 @@ defmodule LightningWeb.Components.Menu do
 
   def projects_dropdown(assigns) do
     ~H"""
-    <div class="relative my-4 mx-2 px-2">
-      <button
-        type="button"
-        class="relative w-full cursor-default rounded-md bg-white py-1.5 pl-3
+    <%= if length(assigns[:projects]) == 0 do %>
```

Comment:
When accessing assigns in Heex templates, please use `@projects` - that way the renderer can optimise for diffs/prevent rerenderers.

The example below, `assigns[:selected_project]` is an exception to the rule since in that specific case we don't know if there is an assign with that key.

But for any other case, we always use the `@name` style.